source/lib/remediation_runbook-stack.ts

```diff
@@ -1363,6 +1363,14 @@ export class RemediationRunbookStack extends cdk.Stack {
       remediationPolicy.addResources('*');
       inlinePolicy.addStatements(remediationPolicy);
 
+      const iamPerms = new PolicyStatement();
+      iamPerms.addActions('iam:GetRole');
+      iamPerms.effect = Effect.ALLOW;
+      iamPerms.addResources(
+        `arn:${this.partition}:iam::${this.account}:role/${remediationRoleNameBase}${remediationName}`
```

Comment:
no change needed: this file is written in the style of crafting policies explicitly. CDK offers another way to manage permissions called grants: https://docs.aws.amazon.com/cdk/v2/guide/permissions.html

I think for new code we should try to use that method since you don't have to craft the whole policy and build the arn pattern, you just say `role.grant(role, 'GetRole');`, I think.